# Cartesian movement message
Header header
position[] queue
int8 speed